### PR TITLE
fix(tagger): products are listed under topics

### PR DESCRIPTION
**Pull request by rofe**

### CODE REVIEW

File: tools/tagger/index.html [hunk 1]
```diff
@@ -136,7 +136,7 @@
         h=d.documentElement,t=setTimeout(function(){h.className=h.className.replace(/\bwf-loading\b/g,"")+" wf-inactive";},config.scriptTimeout),tk=d.createElement("script"),f=false,s=d.getElementsByTagName("script")[0],a;h.className+=" wf-loading";tk.src='https://use.typekit.net/'+config.kitId+'.js';tk.async=true;tk.onload=tk.onreadystatechange=function(){a=this.readyState;if(f||a&&a!="complete"&&a!="loaded")return;f=true;clearTimeout(t);try{Typekit.load(config)}catch(e){}};s.parentNode.insertBefore(tk,s)
 })(document);
 
-
+    var productPaths = ['CC', 'EC', 'DC', 'Other'];
```
Comment (kptdobe) [inline tools/tagger/index.html:139]:
```suggestion
    const productPaths = ['CC', 'EC', 'DC', 'Other'];
```

File: tools/tagger/index.html [hunk 2]
```diff
@@ -179,7 +179,7 @@
         selected.forEach((e) => {
             tagInfo=fetchTag(e);
             html+=`<span data-target="${e}" onclick="remove(this)" class="path active">${tagInfo.path}<span class="tag">${tagInfo.tag}</span></span> `;
-            if (tagInfo.path.toLowerCase().indexOf("products")==0) products+=tagInfo.tag+", ";
+            if (productPaths.includes(tagInfo.path.trim())) products+=tagInfo.tag+", ";
```
Comment (kptdobe) [inline tools/tagger/index.html:182]: You know what you are doing, but the code is doing something completely different and looks weird: why a path would only be CC or EC... ?

Comment (rofe) [reply tools/tagger/index.html:182]: Should be working fine for the moment, but I'll add safeguarding

File: tools/tagger/index.html (hunk 1 above)
Comment (rofe) [reply tools/tagger/index.html:139]: We should lint the entire file 😉

**Pull request merged**